drafts/inlinemacros.md

```diff
@@ -62,16 +65,19 @@ is equivalent to
 
 ## Inline Reductions
 
-The compiler will do the following rewritings.
+The compiler will do the following rewritings when encountering certain patterns of code
+outside the bodies of inline values and inline methods.
+When these patterns of code appear inside the aforementioned constructs, 
+they will be ignored by the inliner and processed by the compiler as usual.
```

Comment:
Actually, as we agreed that in order for method to become `inline` it could need to redo the overloading resolution. So expansion order matters. I would say we need to stick to one decision, and as we decided that expansion happens outside-in than It feels natural to say that inline methods do not expand inside inline methods(otherwise they will lose more precise info that overload resolution could have used).

All this questions of order of expansion seem very subtle for me. I'm sure it would be a source of confusion. And could also become a source of problems, like inlining order in JVM.